docs/BestPractices.md

```diff
@@ -23,6 +23,13 @@ FROM node:4.1.2-onbuild
 RUN groupadd -r node && useradd -r -g node node
 ```
 
+When using the `alpine` variant, add the user like so:
+
+```Dockerfile
+FROM node:7.3.0-alpine
+RUN addgroup -S node && adduser -S -g node node 
+```
```

Comment:
I feel it's important to mention the need for using `USER` later down. I was missing this at first and wondering why my app was still running as root. :-)